src/_pytest/config/argparsing.py

```diff
@@ -355,6 +358,42 @@ def parse_args(self, args=None, namespace=None):
             getattr(args, FILE_OR_DIR).extend(argv)
         return args
 
+    if sys.version_info[:2] < (3, 8):  # pragma: no cover
```

Comment:
I suppose this is enough because we use `-vv` (for example) in our tests, which would blow up without the patch correct?

Also, I thought this fix was not going into 3.8, am I incorrect?

Reply:
Yes, along with -ra and a couple of others... all of which failed when I did the naive version in my last PR!

My upstream PR targets master (3.9), but will almost certainly be backported to 3.8. TBH I think we could merge this now and just fix the condition if the backport somehow falls through!

Reply:
> TBH I think we could merge this now and just fix the condition if the backport somehow falls through!

Agree, just trying to get the full picture. 😁 👍